core/src/main/java/com/opensymphony/xwork2/ognl/OgnlUtil.java

```diff
@@ -194,7 +194,7 @@ private Set<Class<?>> parseClasses(String commaDelimitedClasses) {
             try {
                 classes.add(Class.forName(className));
             } catch (ClassNotFoundException e) {
-                throw new ConfigurationException("Cannot load class for exclusion/exemption configuration: " + className, e);
+                LOG.warn("Class: {} doesn't exist, ignoring it!", className);
```

Comment:
I would strongly advise against this - a ConfigurationException prevents the application from starting when an invalid exclusion list is provided (e.g. accidentally left out a comma or misspelt a class).

If we're going to change this to a warning, I think we need additional validation to prevent mistakes that lead to decreased security.

Alternatively, consider implementing an `optionalExcludedClasses` instead for classes that may not be present in every Struts application.

Reply:
Fair point, I was wondering what was my intention when I had been introducing this change